src/utils/slack.js

```diff
@@ -1,41 +1,28 @@
 const token = process.env.SLACK_TOKEN;
 
-const usernameToID = async username => {
-	let members;
-	await fetch(`https://slack.com/api/users.list?token=${token}`)
+const getUserSlackID = (username, users) => {
+	const user = users.find(
+		user => user.name === username || user.profile.real_name === username
+	);
+	return user ? user.id : null;
+};
+
+const loadUsers = async callback => {
+	fetch(`https://slack.com/api/users.list?token=${token}`)
 		.then(res => res.json())
 		.then(data => {
-			members = data.members;
+			if (!data.ok) callback(null);
+			else callback(data.members);
 		})
-		.catch(err => err);
-
-	if (!members) return null;
-
-	let i = members.length;
-	while (i--) {
-		if (
-			members[i].name === username ||
-			members[i].profile.real_name === username
-		) {
-			return members[i].id;
-		}
-	}
-	return null;
+		.catch(err => callback(null));
 };
 
-const sendSlackMessage = async (username, itemName, storeCode) => {
-	let id = await usernameToID(username);
-	if (!id) {
-		alert('Error: Slack user not found');
-		return null;
-	}
-
-	await fetch(`http://slack.com/api/chat.postMessage?token=${token}&
+const sendSlackMessage = async (id, itemName, storeCode) => {
+	await fetch(`http://slack.com/api/chat.postMessage?token=${
+		this.props.token
+	}&
 	channel=${id}&
-	text=${`Click to purchase your ${itemName}: https://honesty.store/item/${storeCode}`}`)
-		.then(res => alert('Payment reminder sent to Slack'))
-		.catch(error => alert('Error: failed to send reminder'));
-	return true;
+	text=${`Click to purchase your ${itemName}: https://honesty.store/item/${storeCode}`}`);
 };
 
-export default sendSlackMessage;
+export {loadUsers, sendSlackMessage, getUserSlackID};
```

Comment:
Usually named exports have export where they're declared, this makes it easy to see at a glance what's exported

e.g.

```
export const sendSlackMessage
export const loadUsers
```